### ISSUE TITLE
Cannot start ssh agent if no socke file exists previously

### ISSUE THREAD

**Issue by M4RC3L05** [bitwarden/clients#12417]
### Steps To Reproduce

1. Setup `BITWARDEN_SSH_AUTH_SOCK` and `SSH_AUTH_SOCK` to `$XDG_RUNTIME_DIR/bitwarden-ssh-agent.sock`
2. Start bitwarden desktop
3. Check the logs and see the errors:
```
[SSH Agent Native Module] Starting SSH Agent server on "/run/user/1000/bitwarden-ssh-agent.sock"
[SSH Agent Native Module] Could not remove existing socket file: No such file or directory (os error 2)
```

Even tho here i am using env vars to change the socket path, this should break with the default ones.
If i do create the socket file all goes well.

### Expected Result

Bitwarden would create the socket file and path to the socket if the socket file does not exists

### Actual Result

It errors out trying to remove a socket file that is yet to be created.

### Operating System

Linux

### Installation method

Other

### Build Version

2024.12.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by quexten** [comment]
Hi, this bug was already fixed by a community contributor here: https://github.com/bitwarden/clients/pull/12356 and will be included in the next (January) release.

**Comment by M4RC3L05** [comment]
> Hi, this bug was already fixed by a community contributor here: #12356 and will be included in the next (January) release.

Did not know that, thx, closing...
